<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JPEG_converter</class>
 <widget class="QDialog" name="JPEG_converter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="convert_type_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert to:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="file_path">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>721</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="transparency">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Keep Transparency (JPEG does not support transparency)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt; Back</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="convert_type">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Convert">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="height_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Height: </string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="image_view">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>581</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="size_adjustment">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Size Adjustment</string>
   </property>
  </widget>
  <widget class="QLabel" name="width_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Width: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_file">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Select File</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="height">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="width">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="size">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
